src/plugin/tasks/docker/snippets/sls-dynamodb/manage.sh

```diff
@@ -8,7 +8,7 @@ then
     -d amazon/dynamodb-local
   LINK='--link dynamodb-local -e DYNAMODB_LOCAL_ENABLE=true'
 else
-  NET_HOST = '--net host'
+  NET_HOST='--net host'
```

Comment:
Why was this changed?

Reply:
For the variable assignment, it needs to not have spaces or value won't be assigned.

```
var=23: that's the correct variable assignment syntax: a word that consists of unquoted letters, digits or underscores followed by an unquoted = that appears before a command argument (here it's on its own)
```

https://unix.stackexchange.com/questions/258727/spaces-in-variable-assignments-in-shell-scripts